docs/configuredgraphfactory.txt

```diff
@@ -0,0 +1,466 @@
+[[configuredgraphfactory]]
+ConfiguredGraphFactory
+----------------------
+
+The `ConfiguredGraphFactory` is an access point to your graphs, similar
+to the `JanusGraphFactory`.
+
+[[configuredgraphfactory-versus-janusgraphfactory]]
+ConfiguredGraphFactory versus JanusGraphFactory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+However, there is an important distinction between these two graph
+factories:
+
+1.  The `ConfiguredGraphFactory` can only be used if you have configured
+your server to use the `ConfigurationGraphManagement` APIs at server
+start.
+
+The benefits of using the `ConfiguredGraphFactory` is that:
+
+1.  You only need to supply a `String` to access your graphs, as opposed
+to the `JanusGraphFactory`-- which requires you to specify information
+about the backend you wish to use when accessing a graph-- every time
+you open a graph.
+2.  Assuming your `ConfigurationGraphManagement` has not been configured
```

Comment:
Reading this doc from top to bottom, so I don't know what a `ConfigurationGraphManagement` is at this point. I'd suggest introducing all of the concepts in an overview section at the top, before going into depth on each piece below.